research/delf/delf/python/datasets/revisited_op/dataset.py

```diff
@@ -458,12 +462,83 @@ def ReadMetricsFile(metrics_path):
 
     # Parse mean precisions.
     mean_precisions[protocol] = np.array(
-        _ParsePrScores(file_contents_stripped[i + 2], len(pr_ranks)),
-        dtype=float) / 100.0
+      _ParsePrScores(file_contents_stripped[i + 2], len(pr_ranks)),
+      dtype=float) / 100.0
 
     # Parse mean recalls.
     mean_recalls[protocol] = np.array(
-        _ParsePrScores(file_contents_stripped[i + 3], len(pr_ranks)),
-        dtype=float) / 100.0
+      _ParsePrScores(file_contents_stripped[i + 3], len(pr_ranks)),
+      dtype=float) / 100.0
 
   return mean_average_precision, pr_ranks, mean_precisions, mean_recalls
+
+
+def configdataset(dataset, dir_main):
```

Comment:
Can I suggest maybe to rename this function, and explain a little better what it does in the docstring? Like, from the description it's hard to understand what it's returning.